projects/packages/blaze/src/class-blaze.php

```diff
@@ -113,6 +113,12 @@ public static function should_initialize() {
 			$should_initialize = false;
 		}
 
+		// The feature relies on this module for now.
+		// See 1386-gh-dotcom-forge
+		if ( ! ( new Modules() )->is_active( 'json-api' ) ) {
```

Comment:
How does this behave for WPCOM sites?

Reply:
Thanks for the reminder. I had assume it would always return true, but it turns out it doesn't behave like the old `Jetpack::is_module_active()` method.

I've added a wpcom check in 168dd5256f2cacaaa295b64ef3664ac9a10b53a3